src/global/components/footer/Footer.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+import { useMediaQuery } from '@react-hook/media-query';
+import { faTelegram, faInstagram, faFacebook } from '@fortawesome/free-brands-svg-icons';
+import { Logo } from '../Logo';
+import NewsletterSection from './NewsLetterSection';
+import { SocialLinks } from './SocialLinks';
+import { FooterColumn } from './FooterColumn';
+import {useTranslations} from 'next-intl';
+
+export default function Footer() {
+    const isTabletOrMobile = useMediaQuery('(max-width: 768px)');
+    const t = useTranslations('Index');
+
+  const FOOTER_CONTENT={
+    social:[
+     { icon:faFacebook, link:"hliiudfj'j'adsvdfivnv",},
+     { icon:faTelegram, link:"hlihhqa'ewfoafoaf",},
```

Comment:
You can add this [Tele link](https://l.instagram.com/?u=http%3A%2F%2Ft.me%2Fclean_iraq&e=AT3ThC8FJ4-8JtVg-OxxIdLig_JdernzY5ONxIXrYa9OGkBeGhBh6psejsE4H7msdxaagVE9W7UN12GovHJ8ihe4z9dQQPH4w7XMIw)